build/common.mk

```diff
@@ -109,6 +109,10 @@ PULUMI_NODE_MODULES := $(PULUMI_ROOT)/node_modules
 # ensure that `default` is the target that is run when no arguments are passed to make
 default::
 
+dist::
+	$(call STEP_MESSAGE)
+	@if [ -e 'Gopkg.toml' ]; then echo "dep ensure -v"; dep ensure -v; fi
```

Comment:
I assume you are doing this here because you want `make dist` by itself to restore just the golang dependencies (which you'll need to build the other targets).  Is there a reason we can't avoid this and just have the homebrew formula invoke `make ensure dist`?  I assume it is because doing that would cause us to restore pip dependencies which fails on El Capitan?

My personal preference would be to not do this here and either invoke `dep ensure` in the homebrew formula before building or if we could, call make ensure in the formula.

Reply:
I was hesitating between the two. As I just need the go dependencies, I just want to make a dep ensure. This will avoid any problem with python dependencies, make the formula quicker to build and require one less build dependencies (which is great I think).

I can make the dep ensure in the formula without problem.

Reply:
:+1: Thanks!